src/app-root.ts

```diff
@@ -257,6 +262,28 @@ export class AppRoot extends LitElement {
       display: block;
     }
 
+    /* add the following styles to ensure proper modal visibility */
+    body.modal-manager-open {
+      overflow: hidden;
+    }
+    modal-manager {
+      display: none;
+    }
+    modal-manager[mode='open'] {
+      display: block;
```

Comment:
hmm I thought display block was default and that `mode=closed` is where we should display none

Reply:
This is what modal-manager readme.md file shows
```
<style>
  /* add the following styles to ensure proper modal visibility */
  body.modal-manager-open {
    overflow: hidden;
  }

  modal-manager {
    display: none;
  }

  modal-manager[mode='open'] {
    display: block;
  }
</style>
```